src/utils/leetcode_util.py

```diff
@@ -14,6 +16,7 @@ class LeetcodeUtil:
     PROBLEM_URL = "https://leetcode.com/problems/"
     ALL_PROBLEMS_URL = "https://leetcode.com/api/problems/all/"
     DATA_LIMIT = 100
+    TIMEOUT = 10000
 
     def __init__(self):
         pass
```

Comment:
For converting this into a non-static class, all of these properties should be initialized in the `__init__` constructor routine. They should come from arguments, either `kwargs` or positional args. It is okay for them to have default values (in fact they should).

Reply:
Good to know, updated in latest commit.